README.md

```diff
@@ -60,30 +60,30 @@ bedrock.events.on('bedrock-mongodb.ready', function(callback) {
 bedrock.start();
 ```
 
-Below is an example demonstrating the use of a distributed ID generator.
-
-```js
-var database = require('bedrock-mongodb');
-
-database.getDistributedIdGenerator('mynamespace', function(err, idGenerator) {
-  if(err) {
-    console.error('Error', err);
-    return;
-  }
-  idGenerator.generateId(function(err, id) {
-    if(err) {
-      console.error('Error', err);
-      return;
-    }
-    console.log('ID generated', identifier);
-  });
-});
-```
-
 ## Configuration
 
 For documentation on database configuration, see [config.js](./lib/config.js).
 
+### Connecting and Authenticating
+MongoDB's documentation offers tons of great examples on how to authenticate
+using a myriad number of connection strings.
+[Mongo Node 3.5 Driver connect docs](http://mongodb.github.io/node-mongodb-native/3.5/tutorials/connect/)
+[Mongo Node 3.5 Driver atlas docs](https://docs.mongodb.com/drivers/node#connect-to-mongodb-atlas)
+
+You can also connect to access-enabled mongo servers using some small changes to the
+`config.mongodb.connectOptions`:
+```js
+const {connectOptions} = bedrock.mongodb;
+// this optional and only required if connecting to a replicaSet
+connectOptions.replicaSet = process.env.mongo_replicaSet;
+// you must set `ssl` unless `srv` is present in your connection string (in which case, don't set it)
+connectOptions.ssl = true,
+// `auth` should now be used instead of older options `username` and `password`
+connectOptions.auth = { user: process.env.mongo_user, password: process.env.mongo_password },
```

Comment:
with latest revision developers should not need to interact with `auth` at all correct?  If so I suggest that this be removed and replaced with verbiage about using `config.mongodb.username` and config.mongodb.password`

Reply:
I revised the README twice. Are we ready to go here?